spec/controllers/replies_controller_spec.rb

```diff
@@ -229,6 +229,60 @@
       expect(reply.user).to eq(user)
       expect(reply.content).to eq('test content the third!')
     end
+
+    it "allows replies from authors in a closed post" do
+      user = create(:user)
+      other_user = create(:user)
+      login_as(user)
+      reply_post = create(:post, user: other_user, tagging_authors: [user, other_user], authors_locked: true)
+      post :create, params: { reply: {post_id: reply_post.id, content: 'test content!'} }
+      expect(Reply.count).to eq(1)
+      other_post = create(:post, user: user, tagging_authors: [user, other_user], authors_locked: true)
+      post :create, params: { reply: {post_id: other_post.id, content: 'more test content!'} }
+      expect(Reply.count).to eq(2)
+    end
+
+    it "adds authors correctly when a user replies to an open thread" do
+      user = create(:user)
+      login_as(user)
+      reply_post = create(:post)
+      post :create, params: { reply: {post_id: reply_post.id, content: 'test content!'} }
+      expect(Reply.count).to eq(1)
+      expect(reply_post.tagging_authors).to include(user)
+      expect(reply_post.tagging_post_authors.count).to eq(2)
```

Comment:
I'd suggest combining this line and the line above to do `expect(reply_post.tagging_authors).to match_array([reply_post.user, user])`

Reply:
What Throne said